config/jobs/kubernetes/sig-node/sig-node-presubmit.yaml

```diff
@@ -4134,6 +4133,10 @@ presubmits:
         timeout: 90m
       path_alias: k8s.io/kubernetes
       extra_refs:
+        - org: containerd
```

Comment:
the periodic does not use this reference and seems the same code

https://github.com/kubernetes/test-infra/blob/0d7be4fac295635c5761eb9503419ea49020768e/config/jobs/kubernetes/sig-node/node-kubelet.yaml#L890-L899

why is this differnet?

Reply:
https://prow.k8s.io/view/gs/kubernetes-jenkins/logs/ci-kubernetes-node-e2e-cri-proxy-serial/1844218136925048832 failed for 

F1010 03:30:39.884596    9713 gce_runner.go:378] Failed to read metadata file "/home/prow/go/src/github.com/containerd/containerd/test/e2e_node/init.yaml": open /home/prow/go/src/github.com/containerd/containerd/test/e2e_node/init.yaml: no such file or directory


I synced the adding containerd to that job as well.